shared/js/media/jpeg_metadata_parser.js

```diff
@@ -352,8 +352,8 @@ function parseJPEGMetadata(file, metadataCallback, metadataError) {
     case 4: // LONG
       return data.getUint32(offset, byteorder);
     case 5: // RATIONAL
-      return data.getUint32(offset, byteorder) /
-        data.getUint32(offset + 4, byteorder);
+      return [ data.getUint32(offset, byteorder),
+               data.getUint32(offset + 4, byteorder) ];
```

Comment:
Is this so you can display shutter speeds as fractions instead of as decimals.  Could you instead leave this as decimals and do something like:

```
if (speed < 1) speed = "1/" + Math.round(1/speed)
```
But as long as we weren't using any fields of this RATIONAL type before, its okay with me if you make this change if it is really needed.